hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/config/ConfigServerException.java

```diff
@@ -0,0 +1,22 @@
+package org.apache.mesos.hdfs.config;
+
+/**
+ * Indicates a failure to startup the config service, likely a jetty failure.
+ */
+public class ConfigServerException extends RuntimeException {
+
```

Comment:
I don't think we need an empty constructor here. The other constructors look good though.